index.js

```diff
@@ -101,13 +101,29 @@ async function extractFrontMatter(pluginOptions, files, root) {
   debug('start: read all mdx files')
   const fileContents = await Promise.all(files.map(f => fs.readFile(f, 'utf8')))
   debug('finish: read all mdx files')
-  const fmPaths = files.map(f => generateFrontmatterPath(f, root))
+  const fmPaths = files.map(f => generateFrontmatterPath(f, root));
```

Comment:
Formatting is inconsistent with changes - I was using prettier locally to format but I will add enforcement so it's easier when contributing 😀